packages/vscode-extension/package.nls.json

```diff
@@ -274,10 +274,12 @@
   "teamstoolkit.handlers.installOfficeAddinDependencyCancelled": "Dependency installation is canceled, but you can install dependencies manually by clicking the 'Development - Check and Install Dependencies' button on the left side.",
   "teamstoolkit.localDebug.learnMore": "Get More Info",
   "teamstoolkit.localDebug.m365TenantHintMessage": "After enrolling your developer tenant in Office 365 Target Release, enrollment may come into effect in couple of days. Click 'Get More Info' button for details on setting up dev environment to extend Teams apps across Microsoft 365.",
-  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot, install its extension first.",
-  "teamstoolkit.handlers.askInstallCopilot.install": "Install",
-  "teamstoolkit.handlers.askInstallCopilot.cancel": "Cancel",
-  "teamstoolkit.handlers.installExtension.output": "You need to install %s following \"%s\" first.",
+  "teamstoolkit.handlers.askInstallCopilot": "To use GitHub Copilot Extension for Teams Toolkit when developing Teams apps, you need to install both GitHub Copilot and GitHub Copilot Extension for Teams Toolkit (@teamsapp).",
+  "_teamstoolkit.handlers.askInstallCopilot.comment": "@teamsapp is how user invokes Teams agent in GitHub Copilot Chat which should not be translated.",
```

Comment:
@teamsapp is used to activate Teams agent in Github Copilot Chat and should not be translated.